### ISSUE TITLE
loss

### ISSUE THREAD

**Issue by DELTA37** [DELTA37/TimeSeries#8]
Provide a suitable interface for loss function
It must provide backward and forward as a standart torch interface

**Comment by DELTA37** [comment]
loss function we can get from our model, using method get_criterion()
Also in this commit i have fixed bug in train.py related with loss

Commit -  db86c7d
